System/OsString/Common.hs

```diff
@@ -78,15 +166,19 @@ import System.OsPath.Encoding
 import System.IO
     ( TextEncoding, utf16le )
 import GHC.IO.Encoding.UTF16 ( mkUTF16le )
-import qualified System.OsPath.Data.ByteString.Short.Word16 as BS16
-import qualified System.OsPath.Data.ByteString.Short as BS8
+import qualified System.OsPath.Data.ByteString.Short.Word16 as BS
```

Comment:
Would using `BS_8or16` instead of `BS` help with clarity? I don't know the conventions used in `filepath`.

Reply:
I think `BSP` for ByteString Platform is shorter and ok